src/features/board/components/board-issue-row.tsx

```
@@ -0,0 +1,292 @@
+			style={style}
+			{...attributes}
+			{...listeners}
+			className={cn(
+				"group flex items-center gap-2 px-3 py-2 rounded-md hover:bg-muted/60 dark:hover:bg-gray-800/60 cursor-pointer transition-colors duration-100 border border-transparent hover:border-border/40 select-none",
+				isDragging && "opacity-40",
+				isDragOverlay && "shadow-lg bg-background border-border opacity-100"
+			)}
+			onClick={(_e) => {
+				if (!isDragging) onClick();
+			}}
+		>
```

Biome (2.4.4)

[error] Static Elements should not be interactive.

(lint/a11y/noStaticElementInteractions)